Documentation/Experimental/SceneUnderstanding/README_SceneUnderstanding.md

```diff
@@ -0,0 +1,99 @@
+# MRTK Scene Understanding Demo
+
+This is an __experminental__ prelease that integrates [Scene Understanding](https://docs.microsoft.com/en-us/windows/mixed-reality/scene-understanding) (SU) as a MRTK.
+
+SU - in a nutshell, will return a semantic representation of scene entities as well as their geometric forms on __HoloLens 2__.
+
+## Goals and features of the demo
+* Demo scene with visualization of Scene Objects with options for configuring the observer
```

Comment:
I would also emphasize early on that the current experimental feature only works on HL1/HL2 or just HL2 etc